ui/package.json

```diff
@@ -89,6 +89,7 @@
     "shebang-loader": "^0.0.1",
     "snake-case": "^3.0.3",
     "subscriptions-transport-ws": "^0.9.16",
+    "taskcluster-client-web": "^39.2.0",
```

Comment:
This seems OK to me. The client feels like a utility library, why not install it rather than sneak the code in via the monorepo.

Reply:
Cool, thanks.  It feels weird to me that we'll almost always be pulling in an older version of this library (so TC 40.0.0 will use tc-client-web 39.x.y).  But aside from being weird it doesn't really matter.

Reply:
I know I'm late but have you considered NPM's relative syntax?
```json
"taskcluster-client-web": "file:../clients/client-web",
```

That way you get the same behaviour as the versioned import with the one downside that it doesn't always rely on the stable version of the library.

Reply:
I tried that, and tc-client-web does not find its requirements unless `yarn install` has been run in that directory.  I think that's due to Node's (and webpack's) search pattern for `node_modules` -- it never looks in `ui/node_modules`.  I missed this in my first push to this PR because I had a `clients/client-web/node_modules` laying around from development in that directory, but it did not exist in CI.

I tried `link:../clients/client-web` as well, which in theory fixes this -- it makes a link from `ui/node_modules/taskcluster-client-web` -> `../../clients/client-web`.  The `yarn install` process correctly installs taskcluster-client-web's deps in `ui/node_modules`, and iirc from my experimentation Node correctly finds them.  But, webpack does not -- it seems that webpack is using `path.resolve` or similar to translate that symlink into a direct path, and then seeing the same issue as with `file:..`.

It's possible that using a newer version of Webpack would fix this.  It's also possible I am missing some setting or other config that would fix it.  I tried various permutations of aliases with no success.

Reply:
Ah yeah, I forgot that it doesn't recursively install, I've run into that before. What a shame. :neutral_face: